client/src/components/AllEntries.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+
+export default function AllEntries() {
+    fetch("/allEntries")
+    .then((res) => res.json())
+    .then((entry) => {
+      entry.forEach((element) => {
+        // container
+        let Cont = document.createElement("div");
+        Cont.id = element._id;
+        Cont.className = "listing-cont";
+
+        //title element
+        let Title = document.createElement("h3");
+        Title.innerText = element.title;
+        Title.className = "listing-title";
```

Comment:
and this one could just be written out as `<h3 className="listing-title">{element.title}</h3>`